src/popup/router/components/AccountInfo.vue

```diff
@@ -21,18 +26,24 @@
 import { mapGetters } from 'vuex';
 import Copyicon from '../../../icons/copy.svg?vue-component';
 import UserAvatar from './UserAvatar';
+import { getState } from '../../../store/plugins/persistState';
 
 export default {
   components: { Copyicon, UserAvatar },
   data: () => ({
     copied: false,
+    availNames: false,
   }),
   computed: {
     ...mapGetters(['account', 'activeAccountName']),
     chainName() {
       return this.activeAccountName.includes('.chain');
     },
   },
+  async created() {
+    const { names } = await getState();
```

Comment:
I think `getState` is a too low-level method, can we get the same from `this.$store.state`?

Reply:
You are right